native/packages/react-native-bpk-component-pagination-dots/src/BpkPaginationDots-test.common.js

```diff
@@ -23,51 +23,29 @@ import BpkPaginationDots from './BpkPaginationDots';
 
 const commonTests = () => {
   describe('BpkPaginationDots', () => {
-    // Using 10 as an arbitrary high number to test lots of permutations.
```

Comment:
I understand the intention of these tests, however with snapshots this is a misguided strategy, particularly for developers making changes and validating snapshots -> if a developer has 100 snapshots to verify for one change then the chances are they'll miss something which renders the test redundant. Much better to add a few meaningful snapshots and rely on the storybooks for that extra validation.